src/app.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
+/* eslint-disable no-console */
 
-// Write your code here
+const readline = require('readline');
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput');
+const { generateRandomNumber } = require('./modules/generateRandomNumber');
+const { getBullsAndCows } = require('./modules/getBullsAndCows');
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const guessNumber = generateRandomNumber();
+let count = 0;
+
+function playGame(question = '') {
+  terminal.question(question, (userInput) => {
+    if (!checkIsValidUserInput(userInput)) {
```

Comment:
Code quality issue: It's a good practice to keep your functions small and focused on doing one thing. The `playGame` function is doing multiple things: validating the user input, incrementing the count, getting the result, logging the result, checking the result, and deciding what to do next. Consider breaking it down into smaller functions to improve readability and maintainability.